packages/Interface/src/OperatorVector/DTK_MapOperator.hpp

```diff
@@ -123,11 +123,14 @@ class MapOperator : public Tpetra::Operator<double,int,SupportId>
 	Teuchos::ETransp mode = Teuchos::NO_TRANS,
 	double alpha = Teuchos::ScalarTraits<double>::one(),
 	double beta = Teuchos::ScalarTraits<double>::zero() ) const override;
-    bool hasTransposeApply() const override { return false; }
+    bool hasTransposeApply() const override;
     //@}
 
   protected:
 
+    //! Tranpose apply option.
+    virtual bool hasTransposeApplyImpl() const = 0;
```

Comment:
Yes that's what I was looking at. It made more sense for ``apply(...)`` and ``setup(...)`` but consistency is a fine reason. No worries.